services/src/main/java/org/keycloak/services/error/KeycloakErrorHandler.java

```diff
@@ -89,7 +89,9 @@ public static Response getResponse(KeycloakSession session, Throwable throwable)
             OAuth2ErrorRepresentation error = new OAuth2ErrorRepresentation();
 
             error.setError(getErrorCode(throwable));
-            error.setErrorDescription("For more on this error consult the server log at the debug level.");
+            if (statusCode >= 500 && statusCode <= 599) {
```

Comment:
Not sure why we ever decided to use the raw status code and not a `Response.Status`, this could be 'simplified' to:

```suggestion
            Response.Status status = Response.Status.fromStatusCode(statusCode);
            
            if (status.getFamily().equals(Response.Status.Family.SERVER_ERROR)) {
```